python/README.md.in

```diff
@@ -39,13 +39,16 @@ able to use it without any further installation steps.
 ${{ deprecation_notice }}
 
 > **Important:** 
-> Please check if you have an existing installation of the `cuda-quantum`, `cudaq-quantum-cu11`, or `cuda-quantum-cu12` package, and uninstall it prior to installation. Different CUDA-Q binary distributions may conflict with each other causing issues.
+> Please check if you have an existing installation of the `cuda-quantum`, 
+`cudaq-quantum-cu11`, or `cuda-quantum-cu12` package, and uninstall it prior to
+ installation. Different CUDA-Q binary distributions may conflict with each 
+ other causing issues.
 
 
 If you want to perform multi-GPU simulations, additional components must be installed. We
 recommend using [Conda](https://docs.conda.io/en/latest/) to do so. If you are
 not already using Conda, you can install a minimal version following the
-instructions [here](https://docs.anaconda.com/miniconda/). The following
+instructions [miniconda](https://docs.anaconda.com/miniconda/). The following
```

Comment:
This will write "you can install a minimal version following the instuctions [miniconda](https://docs.anaconda.com/miniconda/)" which sounds grammatically incorrect. Perhaps make the hyperlink span a few words, or just keep it as it was.

```suggestion
[miniconda instructions here](https://docs.anaconda.com/miniconda/). The following
```